exercises/object-oriented-programming/src/main/java/com/cbfacademy/accounts/SavingsAccount.java

```diff
@@ -0,0 +1,16 @@
+package com.cbfacademy.accounts;
+
+public class SavingsAccount extends Account {
+    final static private double INTEREST_PERCENTAGE = 6.7;
```

Comment:
This is a creative approach, but we probably want to allow for different accounts to have their their own interest rate set, similarly to how current accounts can have distinct overdraft limits